src/mca/schizo/ompi/schizo-ompi-cli.rst

```diff
@@ -10,8 +10,133 @@
 
    $HEADER$
 
+General command line options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following general command line options are available.
+
+The ``--add-host`` option
```

Comment:
You probably don't want this as `mpirun` cannot extend the DVM since it is a one-shot operation.